src/util/f2n.h

```diff
@@ -45,7 +45,7 @@ class f2n {
         m_manager.set(m_one, ebits, sbits, 1);
     }
 
-    f2n(f2n && other) : m_manager(other.m_manager), m_mode(other.m_mode), m_ebits(other.m_ebits), m_sbits(other.m_sbits),
+    f2n(f2n && other) noexcept : m_manager(other.m_manager), m_mode(other.m_mode), m_ebits(other.m_ebits), m_sbits(other.m_sbits),
```

Comment:
this one can go too.

Reply:
When this one goes away, there's a build error on Ubuntu in CI. I've added it back for now.